<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="pws_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="num_cb">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Использовать цифры</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="punct_cb">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Использовать спецсимволы</string>
    </property>
   </widget>
   <widget class="QLabel" name="gen_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Генератор паролей</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_bar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="gen_display">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pw_len">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество символов</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
